### DISCUSSION TITLE
Http mulaw audio stream not recognized by probe

### DISCUSSION THREAD

**Discussion by TheNetStriker** [datarhei/restreamer#835] [Q&A]
I would like to mux a video and audio stream from my Doorbird camera using Restreamer.

The video format was recognized automatically using the probe, but the audio format over HTTP is not recognized.

I found various sources about this problem and the solution is to add **-f mulaw -ar 8000** to the ffmpeg input, but I don't have the option to add custom options in Restreamer.

The only way I managed to add the audio stream was to add a different audio stream in the UI and then edit the db.json manually to add the **-f mulaw -ar 8000** options.

Is there maybe a better way to do this? The HTTP stream returns the Content Type audio/basic, but that does not seem to help ffmpeg to detect the format automatically.

**Comment by ioppermann** [comment]
We'll see how we can add these options.
